language/tools/move-package/src/resolution/resolution_graph.rs

```diff
@@ -248,7 +251,12 @@ impl ResolvingGraph {
         self.unify_addresses_in_package(&package, &mut resolution_table, is_root_package)?;
 
         let package_path = package_path.canonicalize()?;
-        let source_digest = checksumdir::checksumdir(&package_path.to_string_lossy().to_string())?;
+        // Don't include the build directory in checksums
+        let checksum_options = checksumdir::ChecksumOptions::new(vec!["build"], true, false);
```

Comment:
feels a bit magic. Do we have a shared const for this directory somewhere?

Reply:
ah good point. This is actually `CompiledPackageLayout::Root.path().to_string_lossy()` so will update.